mesos/framework/consumer_task.go

```diff
@@ -0,0 +1,97 @@
+package framework
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+	mesos "github.com/mesos/mesos-go/mesosproto"
+	util "github.com/mesos/mesos-go/mesosutil"
+)
+
+type ConsumerTask struct {
```

Comment:
ConsumerTask should also have `String()` like [this](https://github.com/stealthly/go_kafka_client/blob/mesos-framework/mesos/framework/task.go#L323) so that status calls show task `type: consumer` to understand what is running/added